File: python/customers_paginate.py
import stripe
stripe.api_key = "sk_your_api_key"

customers = stripe.Customer.all(limit=100)

for customer in customers.data:
  print(customer.id)

while customers.has_more:
  customers = stripe.Customer.all(limit=100, starting_after=customers.data[-1])
  for customer in customers.data:
    print(customer.id)

File: ruby/customers_paginate.rb
require 'stripe'

Stripe.api_key = "sk_your_api_key"

customers = Stripe::Customer.all(:limit => 100)

customers.each do |customer|
  puts customer.id
end

while customers.has_more do 
  customers = Stripe::Customer.all(:limit => 100, :starting_after => customers.data.last.id)

  customers.each do |customer|
    puts customer.id
  end
end